lib/Graph/Graph.cpp

```diff
@@ -1299,6 +1299,14 @@ ScatterAssignNode *Function::createScatterAssign(llvm::StringRef name,
   return addNode(new ScatterAssignNode(name, data, indices, slices));
 }
 
+Variable *Function::createShape(llvm::StringRef name, NodeValue input) {
```

Comment:
I am not convinced that this should be in Graph API. Would you use it if you were to create graph using C++ API?